templates/s3-bucket.yaml

```diff
@@ -0,0 +1,94 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Transform: S3Objects
+
+Description: >-
+  An S3 bucket for use in development. This bucket will serve as the landing
+  bucket for incoming data, mimicking the Bridge bucket.
```

Comment:
Change this description so that it's more general. We want to use this template to create buckets in develop and production for storing partly or wholly processed RECOVER data.